checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -41,22 +57,23 @@
     extends GenericAnnotatedTypeFactory<CFValue, LockStore, LockTransfer, LockAnalysis> {
 
     /** Annotation constants */
-    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE;
+    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE, GUARDEDBYINACCESSIBLE, GUARDEDBY, GUARDEDBYBOTTOM, JCIPGUARDEDBY, JAVAXGUARDEDBY, GUARDSATISFIED;
```

Comment:
You don't want them to be subtypes of each other. You want them to be aliases and your logic should only check for the main GuardedBy annotation.